Sources/Network/Service/DefaultServiceProvider.swift

```diff
@@ -22,7 +22,8 @@ final class DefaultServiceProvider: ServiceProvider {
         networkConfig: NetworkConfig,
         targetQueue: DispatchQueue?,
         grpcConnectionFactory: GrpcProtocolConnectionFactory,
-        httpConnectionFactory: HttpProtocolConnectionFactory
+        httpConnectionFactory: HttpProtocolConnectionFactory,
+        rng: MobileCoinRng? = nil
```

Comment:
why do we need to make changes here ? I see that you're adding this parameter but its not used anywhere.

Reply:
Ya, these are part of a bunch of vestigial changes from when I had tested the ChaCha20Rng as a general replacement for the default RNG.  Cleaning them out.